easyconfigs/k/Kaldi/Kaldi-20220213-foss-2021a.eb

```diff
@@ -33,7 +33,7 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('SoX', '14.4.2'),
+    ('SoX', '14.4.2', '', ('GCCcore', '10.3.0')),
```

Comment:
That shouldn't be necessary, as `GCCcore 10.3.0` is a subtoolchain of `foss 2021a`. Did you get an error?